src/sentry/workflow_engine/types.py

```diff
@@ -40,6 +40,7 @@ class WorkflowJob(EventJob, total=False):
     has_alert: bool
     has_escalated: bool
     workflow: Workflow
+    snuba_results: list[int]
```

Comment:
🤔 is the value in this that we can re-use `evaluate_workflows` method?

i think this is a bit of a smell that the abstraction might not be quite right in either delayed processing or the evaluate_workflow_triggers 🤔

mind adding a TODO here so i can come back and take a look? i'm not sure if this is the best approach, but seems okay for now.

Reply:
yeah the value is the we can use at least the `evaluate_workflow_triggers` function. we'll already have processed the actions we can possibly fire before enqueuing so all we need to do is process the slow conditions, but i'm also not sure if it's the best way to do it